### ISSUE TITLE
Post form fields, Require/Optional and What new fields will add next?

### ISSUE THREAD

**Issue by greenlikeorange** [MMAUG/flood-website#23]
Currently is all fields are required by default. Which one we should do optional?
### Donations Group
- title
- description
- phone_numbers
- donation_location
- facebook_url
### News
- title
- description

**Comment by greenlikeorange** [comment]
@swanhtet1992

**Comment by swanhtet1992** [comment]
@setkyar Can you share what you did for the web form Ko Ko Ye requested?

**Comment by setkyar** [comment]
@swanhtet1992 You means the [form](http://dojowithcode.github.io/flood-data//data.html) or [data](https://docs.google.com/spreadsheets/d/1TEsR-GjBV3RgTBN2V3o7INK9npjA4uRNGlhwQOfGQEY/edit?usp=sharing)?

**Comment by setkyar** [comment]
Ah, sorry. I got it.
![enter image description here](https://scontent-kul1-1.xx.fbcdn.net/hphotos-xpt1/v/t34.0-12/11721312_10207966627710412_1373009203_n.jpg?oh=7b19fe2ceef6cee6eb02f9d77c084c31&oe=55C38927)
<br>
![enter image description here](https://scontent-kul1-1.xx.fbcdn.net/hphotos-xfa1/v/t34.0-12/11787326_10207966627830415_1374312424_n.jpg?oh=f86cd1ca5e03fc5cce4aca4756ed9c3a&oe=55C3769D)

**Comment by swanhtet1992** [comment]
## Donation Group
- Group Name
- Target Support
- Target Area
- About
- Facebook link
- Email
- Address
- Phone

## News
- Title
- Content
- Source
- Location
- Category ?

**Comment by greenlikeorange** [comment]
Yep my question is "All fields will mentotary or which are optional?"

**Comment by swanhtet1992** [comment]
အကုန်လိုမယ်.. :D

**Comment by nyanlynnhtut** [comment]
@swanhtet1992 @greenlikeorange Category are dropdown?
